app-backup/kfoldersync/kfoldersync-3.1.0.ebuild

```diff
@@ -25,7 +25,10 @@ RDEPEND="
 	$(add_frameworks_dep kitemviews)
 	$(add_frameworks_dep knotifications)
 	$(add_frameworks_dep kwidgetsaddons)
+	$(add_frameworks_dep kwindowsystem)
 	$(add_frameworks_dep kxmlgui)
+	$(add_qt_dep qtconcurrent)
+	$(add_qt_dep qtcore)
```

Comment:
qtcore is already being pulled in by kde5.eclass, none of the inheriting ebuilds needs to set it ;)